src/julia_gc.cc

```diff
@@ -1030,7 +1322,7 @@ void SwapMasterPoint(Bag bag1, Bag bag2)
 
 // HOOK: mark functions
 
-inline void MarkBag(Bag bag)
+void MarkBag(Bag bag)
```

Comment:
Why?

Reply:
non-static inline functions can produce linker problems and they do in this case. We had a similar issue a while ago (with a hash implementation in another file), and as in practice, `inline` is just treated as a recommendation by optimizers, I removed the keyword.